src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+
+## What are decentralized social networks? {#what-are-decentralized-social-networks}
+Decentralized social media networks are blockchain-based platforms that allow users to exchange information as well as publish and distribute content to audiences. Because these applications run on the blockchain, they are decentralized and resistant to censorship and undue control. 
+
+Many decentralized social networks exist as alternatives to established social media services, such as Facebook, LinkedIn, Twitter, and Medium. But blockchain-powered social networks have a number of features that put them ahead of traditional social platforms. 
+
+## Why do decentralized social media platforms matter? {#decentralized-social-media-benefits}
+
+To understand the benefits of decentralized social media networks, it's important to compare them against traditional social networks:
+
+| Decentralized social media                                                                                                      | Traditional social media                                                                 |
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
+| Users can enjoy privacy in communication.                                                                               | Users are forced to exchange privacy for free access.                                          |
+| Well-designed monetization framework comprising NFTs, in-app crypto payments, and more.                                 | Poor monetization models.                                            
+| Blockchain security protects users from malicious attacks and safeguards data.                                          | Centralization increases the risk of data breaches and identity theft.                          |
+| Platforms are censorship-resistant and open to everyone.                                                                | Individual access is controlled by centralized entities; platforms can be shut down. 
+| Source code and algorithms are fully available for inspection. Users can vote on platform changes in cases.             | Opaque implementation of algorithms. 
```

Comment:
This either needs to be removed or explained elsewhere. Open to how you think it could fit in but it might be getting pretty in the weeds trying to explain governance processes here.

```suggestion
| Source code and algorithms are fully available for inspection.           | Opaque implementation of algorithms. 
```

Reply:
Definitely agree that we can take this one out.